ui/app/AppLayouts/Profile/views/ContactsView.qml

```diff
@@ -117,7 +119,7 @@ SettingsContentBase {
                 enabled: root.contactsStore.receivedContactRequestsModel.count > 0 ||
                          root.contactsStore.sentContactRequestsModel.count > 0
                 text: qsTr("Pending Requests")
-                badge.value: root.contactsStore.receivedContactRequestsModel.count
+                badge.value: root.contactsStore.receivedContactRequestsModel.count + root.contactsStore.sentContactRequestsModel.count
```

Comment:
I thought that it's intentional to notify via a badge only those incoming requests, not counting those created by us. Are you sure it's intended?

Reply:
The same login (only incoming requests are counted) is used for badges in the settings menu and in "messaging" tab:
![image](https://github.com/user-attachments/assets/e54e7db1-cd7c-4596-89ce-f031746ddd7e)

The badge on `Pending requests` tab is consistent with another badges then. If we want to change it, probably all places should be changed. But imo counting only external requests makes sense.

Reply:
Yeah right, I'll revert that hunk

Reply:
@micieslak fixed